### ISSUE TITLE
Saturday OCT 28 Update

### ISSUE THREAD

**Issue by JRDeLisle** [rlh-aagis/vacant2vibrant#34]
I was able to "guess" my PW and get in; we do need a Forgot PW or Reset. Also, need to indicate whether the User Name is not found or PW wrong.
A couple of others:

- I use address search, 123 and pick 123 Topping; it goes to the map, but there is no 123 Topping showing.

- I logged in, entered Home to start over and I am no longer logged in.
![image](https://user-images.githubusercontent.com/8596328/32136893-39396498-bbdc-11e7-9d37-eebe4adf7d42.png)

- After re-logged, I am still on NEIGH boundary; pick a new property it goes to radius; what if I want NEIGH? which some will want. 

- Logout. Probably need a logout; that might help us track how long they were in and make them feel more comfortable that we are not following them. 

- Tracking? Are we tracking their searches?

- Area Stats. Great start, but still not information as not relative. Can we do something like following?
 Show # crimes, etc. to left;
Show bar for the area
Show AVG for KC
![image](https://user-images.githubusercontent.com/8596328/32137000-9702dfea-bbdd-11e7-8913-10cc0d80ff5c.png)

- If pick a house Registered after viewing NEIGH stats, not see the bar chart

- Saved Searches. I'd love to see how this works? When someone logs back in, will we say, Welcome Jim. Want to pick up where left off or recall search?  Will they be able to Name Searches?

- Property Details
County Parcel Number; some may know this is a link, but others not. How about Link to County Data?
Sold / Available change to Sold/Unsold

Sorry if these are in other issues...

**Comment by rlh-aagis** [comment]
Display Market (KCMO) average for each quintile, display actual number rather than quintile value along with average (red line).

**Comment by JRDeLisle** [comment]
Yes; that will be powerful info. Ron, did we get the averages for the market as a whole for benchmarking?

**Comment by andreiradulescu** [comment]
Added in the feature to record user searches.
